pyrokinetics/local_geometry/LocalGeometryMXH.py

```diff
@@ -190,6 +189,19 @@ def from_local_geometry(
             local_geometry=local_geometry, verbose=verbose, show_fit=show_fit
         )
 
+    @classmethod
+    def from_gk_data(cls, params: Dict[str, Any], n_moments: Optional[int] = 4):
+        """
+        Initialise from data gathered from GKCode object, and additionally set
+        bunit_over_b0
+        """
+
+        cls.n_moments = n_moments
```

Comment:
This has the potential to break things. If two `LocalGeometryMXH`'s are created in the same Python session, and they each set `n_moments` to something different, they'll both share the value assigned to whichever was created last, as the attribute is set on the class rather than an instance of it. As `n_moments` needs to passed in before the call to `from_gk_data`, the only solution I can think of is to update `from_gk_data` to take optional `**kwargs` and to pass in `n_moments` via that.

(Alternatively, if we're confident that `n_moments == 4` everywhere, it could just be removed as an optional argument)

Reply:
CGYRO/TGLF can only use 4 moments so there's actually no need to have this as an option. We should force `n_moments=4` for all `MXH` geometries